ftw/upgrade/executioner.py

```diff
@@ -49,6 +49,27 @@ def install_upgrades_by_api_ids(self, *upgrade_api_ids):
         data = [(upgrade['profile'], [upgrade['id']]) for upgrade in upgrades]
         return self.install(data)
 
+    security.declarePrivate('install_profiles_by_profile_ids')
+    def install_profiles_by_profile_ids(self, *profile_ids):
+        for profile_id in profile_ids:
+            # Starting from GenericSetup 1.8.0 getLastVersionForProfile can
+            # handle profile ids with or without 'profile-' prefix, but we need
+            # to support older versions as well, which only support it without
+            # the prefix.
+            prefix = 'profile-'
+            if profile_id.startswith(prefix):
+                profile_id = profile_id[len(prefix):]
+            installed = self.portal_setup.getLastVersionForProfile(profile_id)
+            if installed != 'unknown':
+                logger.info('Ignoring already installed profile %s.',
+                            profile_id)
+                continue
```

Comment:
Should there be a way to reinstall an already installed profile?
The `continue` here makes it impossible to reinstall a profile which was already installed.
It is also not possible to uninstall a profile, since `portal_setup` does not support uninstalling (only quickinstaller does).

I'm not sure what we want to do. I see these options:

1. Do not change the behavior: we are simply not (yet) supporting reinstalling profiles.
2. Remove the installed check: we do not care whether the profile is already installed; the user should know what he's doing (+1)
3. Add a `--force` argument for changing the behavior (-1)

If we keep the current behavior (1.), I'd not only ask `portal_setup` but also `portal_quickInstaller`, so that it is possible to uninstall a product in quickinstaller (manually) and then reinstall it here. See the method in the [`gatherer`](https://github.com/4teamwork/ftw.upgrade/blob/master/ftw/upgrade/gatherer.py#L214-L228) (and #55).

@mauritsvanrees @lukasgraf what are your thoughts here?